src/Usuarios.php

```diff
@@ -10,9 +10,50 @@ public function testIt($bool = true)
         return $bool;
     }
     
-    public static function createUsuario()
-    {
-        // TODO: Crear usuario nuevo con los datos enviados en el formulario
+    public static function createUsuario(
+        \Psr\Http\Message\ServerRequestInterface $peticion,
+        \Psr\Http\Message\ResponseInterface $respuesta,
+        $argumentos
+    ) {
+        /* Configuramos el tipo MIME correcto para una salida JSON */
+        $respuesta = $respuesta->withHeader('Content-Type', 'application/json');
+        $body = $respuesta->getBody();
+        try {
+            /* Creamos el usuario usando los datos enviados en el formulario */
+            $conexion = \miPDO\Conexion::obtenerPDO();
```

Comment:
__[phpmd]__
* Avoid using static access to class '\miPDO\Conexion' in method 'createUsuario'.